test-suite/js/src/test/scala/org/scalajs/testsuite/library/JsBigIntTest.scala

```diff
@@ -0,0 +1,137 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.testsuite.library
+
+import org.junit.Assert._
+import org.junit.Test
+import org.scalajs.testsuite.utils.JSAssert.assertJSUndefined
+
+import scala.scalajs.js
+
+class JsBigIntTest {
+
+  // Methods we actually implement
+
+  @Test def apply(): Unit = {
+    val bi = js.BigInt("9007199254740992")
+    assertEquals(bi.toString(), "90071992547409912")
+  }
+
+  @Test def asIntN(): Unit = {
+    val x = js.BigInt(2) ** js.BigInt(64)
```

Comment:
I am really surprised that this could work at all on your side. `**` is not supported as a JS operator at the moment. How did this happen?

Reply:
`**` is [exponentiation operator in JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators#Exponentiation).
```
2 ** 3 === 8
2n ** 3n === 8n
```

Reply:
I know that `**` is an ECMAScript operator. But it's not supported by Scala.js. Maybe it should, especially if there is no other way to perform `BigInt` exponentiation. But right now it doesn't, so how did this test pass for you locally at all?